src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -199,40 +176,41 @@ public int compare(@Nullable JavaFunction<?> o1, @Nullable JavaFunction<?> o2) {
 				assert false;
 				throw new NullPointerException();
 			}
-			
+
 			return o1.getName().compareTo(o2.getName());
 		}
-		
+
 	}
-	
+
 	private static final FunctionComparator functionComparator = new FunctionComparator();
 
 	/**
 	 * Generates documentation using template and output directories
 	 * given in the constructor.
 	 */
+	@Override
 	@SuppressWarnings("unchecked")
 	public void generate() {
-		for (File f : template.listFiles()) {
+		for (File f : templateDir.listFiles()) {
 			if (f.getName().matches("css|js|assets")) { // Copy CSS/JS/Assets folders
 				String slashName = "/" + f.getName();
-				File fileTo = new File(output + slashName);
+				File fileTo = new File(outputDir + slashName);
```

Comment:
```suggestion
				File fileTo = new File(outputDir, file.getName());
```
doesn't this work? this way you don't need slashName